### PR TITLE
Postgresql/generate csv

### PR DESCRIPTION
**Pull request by Busyreadingsomething**
[POSTGRES INTEGRATION](https://trello.com/c/KGvN3bFf/2-postgresql-integration-m)
[Design Schema](https://trello.com/c/XSfzLcIJ/8-create-postgresql-schema-s)
[Create Generators](https://trello.com/c/mFOjHs9Q/9-create-psql-data-generators-s)
[Seed DB](https://trello.com/c/dmayY0Fb/10-seed-database-m)

### CODE REVIEW

File: SDC_DB/createSchema.js [hunk 1]
```diff
@@ -75,6 +75,6 @@ function write10Million(start = 1e7) {
     const command = 'mongoimport -d silverspoon -c restaurantmenus --file db/testData.json --type json --numInsertionWorkers 4';
```
Comment (Angelhendrix) [inline SDC_DB/createSchema.js:75]: I added 18 workers you are being too modest lol - no change needed

File: SDC_DB/postgresDB/seedDB.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+CREATE DATABASE silverspoon;
+
+\c silverspoon;
+
+BEGIN;
+
+CREATE TABLE restaurants(
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(50) NOT NULL
```
Comment (Angelhendrix) [inline SDC_DB/postgresDB/seedDB.sql:11]: in psql varchar actually slows it down use text for unlimited text space

**Comment by Angelhendrix** [comment]
everything looks good

File: SDC_DB/createSchema.js (hunk 1 above)
Comment (MichaelDurfey) [reply SDC_DB/createSchema.js:75]: 18?!? I need to get on that level

File: SDC_DB/postgresDB/seedDB.sql [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+);
+
+\copy items (name, menu, price, res_id) FROM '/Users/m.azarshahy/Desktop/SDC/menu-components/SDC_DB/postgresDB/testData/items.csv' DELIMITER ',' CSV;
+
+COMMIT;
+
+BEGIN;
+
+CREATE TABLE tags(
+  id SERIAL PRIMARY KEY NOT NULL,
```
Comment (MichaelDurfey) [inline SDC_DB/postgresDB/seedDB.sql:35]: To help speed up seeding you could try adding the primary key constraint after your data has been seeded

**Review by MichaelDurfey — APPROVED** [review]
Looks good, as always. One suggestion

**Pull request merged**